buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -204,9 +204,10 @@ object Dependencies {
     }
 
     object Aws {
-        private const val VERSION = "1.12.582"
-        private const val VERSION2 = "2.21.16"
+        private const val VERSION = "1.12.593"
+        private const val VERSION2 = "2.21.27"
         private const val JAXB_VERSION = "2.3.1"
+        private const val KOTLIN_SQS_VERSION = "0.35.1-beta"
```

Comment:
Nice. I remember seeing this library before but at the time it was "alpha", so I stayed away.

Reply:
I figure it can't make things work any worse so it's worth a shot to see if it helps.

Reply:
> I figure it can't make things work any worse so it's worth a shot to see if it helps.

Lol